src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -80,19 +78,20 @@ object Content {
    * @param hadoopConfiguration Hadoop configuration.
    * @param pathFilter Filter for accepting paths. The default filter is picked from spark
    *                   codebase, which filters out files like _SUCCESS.
-   * @param throwIfNotExists Throws FileNotFoundException if path is not found. Else creates a
-   *                         blank directory tree with no files.
    * @return Content object with Directory tree starting at root, and containing all leaf files
-   *         from "path" argument.
+   *         from "path" argument. If the given path does not exist, return empty Directory.
```

Comment:
you are returning Content with empty directory?

Reply:
revised